docs/organizations/using-coding-standards.md

```diff
@@ -91,13 +90,10 @@ To edit an existing coding standard or change the repositories that follow that
 
 1.  Click the button **Save and apply standard** on the repository selection page to save your changes to the coding standard.
 
-    !!! important
-        If you stop applying a coding standard to any repository, Codacy restores the previous code pattern configurations for that repository.
-
     Codacy will start using the updated coding standard on the next analysis of each selected repository.
 
-!!! tip
-    To ensure that all new repositories automatically follow a coding standard, [set the coding standard as default](#set-default).
+    !!! note
+        If you stop applying a coding standard to any repositories, Codacy restores the previous code pattern configurations for those repositories. However, the repositories keep the toggle status of the tools from the coding standard.
```

Comment:
There are some tough concepts to convey here!

* "Toggle status" is a bit unclear; I was unsure what it referred to. How about "activation status"?
* I'd use the singular. Personal preference perhaps, but it feels more readable to me.
* I'd also reword to have one subject, (Codacy does X, but also Y) instead of two (Codacy does X, the repositories do Y).

```suggestion
    !!! note
        If you stop applying a coding standard to a repository, Codacy restores the previous code pattern configurations of that repository, but keeps the activation status of the tools as defined by the coding standard.
```

Reply:
These are nice edits for simplifying the text! :star_struck: